handlers/bootstrap.go

```diff
@@ -130,6 +130,17 @@ func buildContainerConfig(containerCmd []string, machine *client.Machine, imgRep
 
 	volConfig := map[string]struct{}{"/var/run/docker.sock": {}}
 	envVars := []string{"CATTLE_PHYSICAL_HOST_UUID=" + machine.ExternalId}
+	for key, value := range machine.Labels {
+		hostLabel := ""
+		switch value.(type) {
```

Comment:
Do we need to actually check for type?  Isn't it already strongly typed as a string when it's passed by the go-rancher client due to the fact the label is a map[string] on the cattle schema?  Or can the value be of other types?  Either way, wouldn't you simply output it as a string regardless in the env variable?